apps/jenkins/jenkins/sbox-intsvc/jenkins.yaml

```diff
@@ -122,7 +122,7 @@ spec:
                 azure:
                   cacheDuration: 3600
                   clientId: "7e2f8c93-7d53-4668-bd84-f134d67dee09"
-                  clientSecret: "$${jenkins-client-secret}"
+                  clientSecret: "${jenkins-client-secret}"
```

Comment:
this should be removed from sbox as you've added the same config to the base now